src/commands/api-mesh/__tests__/create.test.js

```diff
@@ -230,6 +237,8 @@ describe('create command tests', () => {
 		  },
 		  "mesh": {
 		    "meshConfig": {
+		      "meshId": "dummy_id",
+		      "meshURL": "",
```

Comment:
I'm somewhat unclear on what the runResult consists of. Is it just the return value from the command? When is the`meshURL` added to it, considering that it's not part of the `createCommand`?

Reply:
I updated the snapshot and because of wrong config in create command it came under meshConfig